src/test/java/com/google/firebase/auth/FirebaseTokenVerifierImplTest.java

```diff
@@ -399,20 +399,6 @@ public void testVerifyTokenMissingTenantId() {
     }
   }
 
```

Comment:
Why was this removed?

Reply:
Sorry, I forgot to add a comment here.

This test passes, but only because the `verifyToken` does not throw. The current behaviour is that the token verifier silently ignores the `tenantId` in the input token, if the verfier itself is not built with a `tenantId` set.

I was wondering if this is the intended behavior or if we expect the verifier to throw (in which case I can update the implementation and add a `fail()` in the test's try block). 

I haven't tested other tests in this file, but since they lack a `fail()` in the try block, I suspect that there might be a few that are passing regardless of the test conditions.

Reply:
The verifier is initialized with a tenantId in this case if I'm not mistaken:

```java
fullyPopulatedBuilder()
        .setTenantId("TENANT_ID") // <----
        .build()
        .verifyToken(tokenFactory.createToken());
```

We should be able to add `fail()` statement to the `try` block.

Reply:
I'm sorry, I don't get it. This test here does not have a verifier with a tenantId.

```
fullyPopulatedBuilder()
        .build()
        .verifyToken(createTokenWithTenantId("TENANT_ID"));
```


This test is same as the `testVerifyTokenWithTenantId` but it's expecting a different behaviour.

Reply:
This is testing a regular token against a tenant-aware verifier (this should throw):

```
fullyPopulatedBuilder()
        .setTenantId("TENANT_ID")
        .build()
        .verifyToken(tokenFactory.createToken());
```

This is testing a tenant-scoped token against a tenant-unaware verifier:

```
fullyPopulatedBuilder()
        .build()
        .verifyToken(createTokenWithTenantId("TENANT_ID"));
```

This used to throw the same error, but that behavior was changed in #475. This test case seems like something left over from before the change. We can update it to reflect the current behavior.